src/libraries/Directory.Build.props

```diff
@@ -329,4 +329,9 @@
     <CLSCompliant Condition="'$(CLSCompliant)' == ''">true</CLSCompliant>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <ILLinkTrimAssemblyArtifactsRootDir>$(ArtifactsBinDir)ILLinkTrimAssembly/$(BuildSettings)</ILLinkTrimAssemblyArtifactsRootDir>
+    <ILLinkTrimAssemblySuppressionsXmlsDir>$(ILLinkTrimAssemblyArtifactsRootDir)/suppressions-xmls/</ILLinkTrimAssemblySuppressionsXmlsDir>
```

Comment:
Once you use NormalizeDirectory for the property above remove the first `/` and change the trailing one for `\`